sb600spi.c

```diff
@@ -748,13 +760,16 @@ int sb600_probe_spi(const struct programmer_cfg *cfg, struct pci_dev *dev)
 
 	if (((tmp >> 22) & 0x1) == 0 || ((tmp >> 23) & 0x1) == 0) {
 		msg_perr("ERROR: State of SpiAccessMacRomEn or SpiHostAccessRomEn prohibits full access.\n");
```

Comment:
This is still printed for Yangtze++, is this intentional?

Reply:
Yes. I wanted to keep it so that user or developer is informed what is going on.

Reply:
Still, `ERROR` in output may scare users away. Maybe add another line printed for Yangtze that it can be ignored, if you don't want to change this one into something less scary.

Reply:
Flashrom has a lot of scary prints... And if not here, the error will appear later, because if any of those bits is cleared, reads will fail on certain offsets

Reply:
```suggestion
		msg_perr("State of SpiAccessMacRomEn or SpiHostAccessRomEn prohibits full access.\n");
```

msg_perr is already an error print, so I think it is fine...